index.php

```diff
@@ -11,19 +10,25 @@
 			Typekit.load();
 		} catch ( e ) {
 		}</script>
-	<script src="https://maps.googleapis.com/maps/api/js"></script>
-	<?php if ( defined( 'HM_DEV' ) ) : ?>
+	<script src="https://maps.googleapis.com/maps/api/js&key=<?php if ( defined( 'HM_GOOGLE_API_KEY' ) ) echo HM_GOOGLE_API_KEY ?>"></script>
+	<?php if ( defined( 'HM_DEV' ) && HM_DEV ) : ?>
 		<script src="//localhost:35729/livereload.js"></script>
 	<?php endif; ?>
 	<?php do_action( 'opengraph' ); ?>
+	<?php wp_head(); ?>
 </head>
 
-<body>
-<!-- Start Sign-up Box -->
+<body <?php body_class(); ?>>
+<noscript><p style="color: black;">A Day of Rest Boston is built in JavaScript. If you are reading this message it means
```

Comment:
Yes, although im struggling to work out what the best way to i8n it is. There are two paragraphs on that line so it should be two strings to translate but. the URLs in the middle of them is making me :/. about it all.